run_bot_streaming.sh

```
@@ -0,0 +1,209 @@
+
+# Check if Docker is available
+check_docker
+
+# Main function to run the bot
+run_bot() {
+    print_info "${ICON_STREAM} Starting bot with audio streaming enabled..."
+    print_info "📊 Audio will be streamed to: $WEBSOCKET_URL"
+    echo ""
+
+    # Read and process config
+    local output_dir=$(create_output_dir)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)